### PR TITLE
http icon for tools making HTTP requests

### PR DESCRIPTION
**Pull request by trieloff**
- http (icon simply says HTTP) for curl, wget and httpie
(some of the apps that are sometimes running in background tabs for me)

### CODE REVIEW

**Comment by dfrankland** [comment]
`curl` has its own logo, so I am not sure about including it. I prefer to use logos to differentiate, how about you?

File: src/mapIcons.js [hunk 1]
```diff
@@ -21,6 +21,11 @@ module.exports = {
     'git-remote-http',
     'git-remote-https',
   ],
+  http: [
+  	'curl',
+  	'wget',
+  	'http',
```
Comment (dfrankland) [inline src/mapIcons.js]: Was this supposed to be `httpie`?

Comment (trieloff) [reply src/mapIcons.js]: The command for httpie is http – it's a bit confusing.

**Comment by trieloff** [comment]
Yes, we can leave curl it's own logo.

**Comment by trieloff** [comment]
Curl has its own icon now

**Pull request merged**